src/components/VariableEditor.js

```diff
@@ -29,6 +29,7 @@ export class VariableEditor extends React.Component {
     value: PropTypes.string,
     onEdit: PropTypes.func,
     onHintInformationRender: PropTypes.func,
+    onPrettifyQuery: PropTypes.func,
```

Comment:
My reasoning was that since there is a shortcut to run the query here as well, it wouldn't hurt to have the same mechanism for prettifying, and would make for consistent UX. What do you think?